simtools/model/model_utils.py

```diff
@@ -37,19 +42,19 @@ def compute_telescope_transmission(pars, off_axis):
     return pars[0] / (1.0 + pars[2] * t ** pars[4])
 
 
-def is_two_mirror_telescope(telescope_model_name):
+def is_two_mirror_telescope(telescope_model_name: str) -> bool:
     """
-    Check if the telescope is a two mirror design.
+    Determine if the telescope model is a two-mirror telescope.
 
     Parameters
     ----------
     telescope_model_name: str
-        Telescope model name (ex. LSTN-01).
+        Name of the telescope model.
 
     Returns
     -------
     bool
-        True if the telescope is a two mirror one.
+        True if it is a two-mirror telescope, False otherwise.
```

Comment:
As booleans have only two states, it is probably enough to describe the Trues state (no need of `False otherwise`)